yt_dlp/extractor/boosty.py

```diff
@@ -162,9 +165,22 @@ def _extract_formats(self, player_urls, video_id):
 
     def _real_extract(self, url):
         user, post_id = self._match_valid_url(url).group('user', 'post_id')
+
+        auth_cookie = self._get_cookies('https://api.boosty.to/').get('auth')
+        auth_headers = {}
+        if auth_cookie is not None:
+            try:
+                auth_data = json.loads(compat_urllib_parse_unquote(auth_cookie.value))
+                auth_headers['Authorization'] = f'Bearer {auth_data.get("accessToken")}'
```

Comment:
This should brobably be `auth_data["accessToken"]` since we catch `KeyError` below.

Reply:
Ah yes, that's what i wanted. It probably shouldn't happen anyway but I'll change it.